src/components/partition-map/partition-map.component.ts

```diff
@@ -10,7 +10,11 @@ import { ColorService, ThemeColor } from '../../services/color/index';
 @Component({
     selector: 'ux-partition-map',
     templateUrl: './partition-map.component.html',
-    changeDetection: ChangeDetectionStrategy.OnPush
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        'role': 'tree',
+        '[attr.aria-orientation]': 'vertical',
```

Comment:
This is trying to bind to a property named `vertical`, which doesn't exist. I think this should work but you'll need to check:
```suggestion
        'aria-orientation': 'vertical',
```